src/main/java/com/google/api/generator/gapic/composer/SampleCodeHelperComposer.java

```diff
@@ -0,0 +1,160 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.api.generator.gapic.composer;
+
+import com.google.api.generator.engine.ast.AssignmentExpr;
+import com.google.api.generator.engine.ast.CommentStatement;
+import com.google.api.generator.engine.ast.LineComment;
+import com.google.api.generator.engine.ast.MethodInvocationExpr;
+import com.google.api.generator.engine.ast.TryCatchStatement;
+import com.google.api.generator.engine.ast.TypeNode;
+import com.google.api.generator.engine.ast.Variable;
+import com.google.api.generator.engine.ast.VariableExpr;
+import com.google.api.generator.gapic.model.Method;
+import com.google.api.generator.gapic.model.MethodArgument;
+import com.google.api.generator.gapic.model.ResourceName;
+import com.google.api.generator.gapic.utils.JavaStyle;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+public final class SampleCodeHelperComposer {
+
+  public static TryCatchStatement composeRpcMethodSampleCode(
+      Method method,
+      List<MethodArgument> arguments,
+      TypeNode clientType,
+      Map<String, ResourceName> resourceNames) {
+    // Paged Unary RPC method.
+    if (method.isPaged()) {
+      return composePagedUnaryRpcMethodSampleCode(method, arguments, clientType, resourceNames);
+    }
+    // Long-running operation Unary RPC method.
+    if (method.hasLro()) {
+      return composeLroUnaryRpcMethodSampleCode(method, arguments, clientType, resourceNames);
+    }
+    // Pure Unary RPC method.
+    return composeUnaryRpcMethodSampleCode(method, arguments, clientType, resourceNames);
+  }
+
+  public static TryCatchStatement composeRpcDefaultMethodSampleCode(
+      Method method, TypeNode clientType, Map<String, ResourceName> resourceNames) {
+    // TODO(summerji): compose sample code for unary default rpc method.
+    VariableExpr clientVarExpr = createVariableExpr(getClientName(clientType), clientType);
+    String content =
+        String.format(
+            "Note: Not implemented yet, placeholder for unary %s rpc method sample code.",
```

Comment:
I would still prefer not having this comment in all the sample code, in case this makes it into a generated client.

On that note, consider using another branch as the working `master` branch, so that we don't accidentally include a partial sample code implementation in a release.

Reply:
1. "I would still prefer not having this comment in all the sample code, in case this makes it into a generated client."  The PR 506 is outline pr, if not comment here, anything suggest to put here?

Reply:
"On that note, consider using another branch as the working master branch, so that we don't accidentally include a partial sample code implementation in a release."

I got the indention and agree on it. But how can we do that? I haven't done that before.